src/twisted/conch/ssh/userauth.py

```diff
@@ -306,6 +314,45 @@ def auth_publickey(self, packet: bytes) -> Deferred[_ConchPortalTuple]:
             )
         return result
 
+    def _isSecurityKey(self, pubKey):
```

Comment:
We can keep this helper here... no big deal... but since this method is not using the `self` argument, this looks like something is not quite right with the design of this method.

My preference, is to have this code at `twisted.consh.ssh.keys.Key.isSecurtiyKey()`

------

This comment applies for the other new helper methods.

```suggestion
    @staticmetod
    def _isSecurityKey(pubKey):
```